src/quo/components/wallet/approval_label/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo.components.wallet.approval-label.view
+  (:require [oops.core :refer [oget]]
+            [quo.components.buttons.button.view :as button]
+            [quo.components.icon :as icon]
+            [quo.components.markdown.text :as text]
+            [quo.components.wallet.approval-label.schema :as approval-label.schema]
+            [quo.components.wallet.approval-label.style :as style]
+            [quo.foundations.colors :as colors]
+            quo.theme
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [schema.core :as schema]
+            [utils.i18n :as i18n]))
+
+(def ^:private status-icons
+  {:approve   :i/alert
+   :approving :i/pending-state
+   :approved  :i/check})
+
+(def ^:private status-message
+  {:approve   :t/approve-amount-symbol
+   :approving :t/approving-amount-symbol
+   :approved  :t/approved-amount-symbol})
+
+(defn- view-internal
+  [{:keys [status customization-color token-value token-symbol
+           container-style button-props]
+    :or   {customization-color :blue}}]
+  (let [theme                                 (quo.theme/use-theme)
+        [container-width set-container-width] (rn/use-state 0)
+        on-layout                             (rn/use-callback #(set-container-width
+                                                                 (oget % :nativeEvent :layout :width)))]
+    [hole-view/hole-view
+     {:holes     [{:x                       0
+                   :y                       0
+                   :height                  24
+                   :width                   container-width
+                   :borderBottomStartRadius 16
+                   :borderBottomEndRadius   16}]
+      :style     {:margin-top -24}
```

Comment:
What is `-24`? Can we define this number in some constant?

Reply:
24 is the height of the hole-view, and if we don't add the negative margin here when using the component, it will look like this:

<img width="507" alt="Screenshot 2024-05-21 at 09 42 20" src="https://github.com/status-im/status-mobile/assets/19279756/a0b6e153-e17c-4f27-8f13-07378b1422bb">

Instead we want this:
<img width="514" alt="Screenshot 2024-05-21 at 09 42 13" src="https://github.com/status-im/status-mobile/assets/19279756/4a44010e-890b-405d-8039-2c16a968d237">

I have moved the value to a constant.